test/functional/page_objects/visual_builder_page.ts

```diff
@@ -727,4 +727,11 @@ export class VisualBuilderPageObject extends FtrService {
       await this.comboBox.set('topHitOrderByFieldSelect', timeField);
     });
   }
+
+  public async setFilterRatioOption(optionType: 'numerator' | 'denominator', query: string) {
+    const optionInput = await this.find.byCssSelector(
+      `[data-test-subj="${optionType}Row"] [data-test-subj="queryInput"]`
```

Comment:
@VladLasitsa  @DziyanaDzeraviankina 
 we are trying to avoid using css selectors especially if for both components we already added `data-test-subj` attributes  

Please see next example: 
![image](https://user-images.githubusercontent.com/20072247/125963181-71da72d8-1ee4-4e0c-8bf5-b431cd551ecc.png)